apps/web/src/stores/editor-store.js

```diff
@@ -145,6 +147,10 @@ class EditorStore extends BaseStore {
         attachmentsLength: db.attachments.ofNote(note.id, "all")?.length || 0
       };
     });
+    if (!db.notes.note(noteId))
+      this.set((state) => {
+        state.session.readonly = true;
```

Comment:
This approach won't work with deleted notes. We want to make sure of a few things:

1. User should not be able to change anything about the note. The `readonly` property can be changed from properties.
2. We need to show a notice that clearly indicates that this is a deleted note. This notice should also have a restore button. Pressing this button will close the note & restore it from trash.
3. The only information the user should be able to see is the metadata of a note. They shouldn't be able to change any options, see the session history etc.

To that end, it might be a better idea to add a `isDeleted` property in the session which can then be used later on for all the purposes mentioned above.